### ISSUE TITLE
syntex is not maintained

### ISSUE THREAD

**Issue by mmrath** [Dushistov/rust_swig#26]
Just came across your project from SO. Good work. 
Thought I would let you know that syntex is no longer maintained. https://github.com/dtolnay/syn is an alternative

**Comment by Dushistov** [comment]
@mmrath 

Yeah I know, I suppose to solve this issue in the next milestone after [0.1.0 milestone](https://github.com/Dushistov/rust_swig/issues?q=is%3Aopen+is%3Aissue+milestone%3A0.1.0) .
But at now I not sure how, because of further to parser `rust_swig` requires also 
macros expander and compiler like error reporting.

In any case, this issue will be definitely solved  in the `0.2.0`.
